pythonforandroid/recipes/av/__init__.py

```diff
@@ -5,11 +5,15 @@
 class PyAVRecipe(CythonRecipe):
 
     name = "av"
-    version = "10.0.0"
+    version = "13.0.0"
     url = "https://github.com/PyAV-Org/PyAV/archive/v{version}.zip"
 
     depends = ["python3", "cython", "ffmpeg", "av_codecs"]
     opt_depends = ["openssl"]
+    patches = ['patches/compilation_syntax_errors.patch']
+
+    def prebuild_arch(self, arch):
+        self.apply_patches(arch)
```

Comment:
I think this is pretty much what's happening already, so we can probably remove that override and let it apply the patches automatically.
https://github.com/kivy/python-for-android/blob/v2024.01.21/pythonforandroid/build.py#L495-L497

Reply:
That makes sense, removed!